### ISSUE TITLE
Incorrect Unirule mapping: UR000166233 GO:0016192 vesicle-mediated transport

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#4897]
**UniProt UniRule (ID and label):**

UR000166233 GO:0016192 vesicle-mediated transport

**Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPBC839.14c  Q9P7Z3

from
https://www.yeastgenome.org/locus/S000001326
based on a phenotype so could be indirect (tef1 is currently only recorded substrate)

[Q9P7Z3](https://www.uniprot.org/uniprot/Q9P7Z3)

**Description of issue**

**Comment by ValWood** [comment]
Also
Nomenclature history: YIL064W was originally named SEE1 (Secretion and Early Endocytosis) based on defects in both secretion of heterologous proteins and endocytosis. The name was changed to EFM4 (Elongation Factor Methyltransferase) in September 2014 based on a request received from the community. The change unifies the nomenclature for the family of translation factor methyltransferase genes (EFM1, 2, 3, and 4) that encode enzymes which methylate elongation factors.

probably SGD should consider changing GO BP annotation @edwong57 ?

**Comment by PedroRaposo** [comment]
@ValWood I am not sure I understand the issue. The GO:0016192 is wrongly annotated unto Q9P7Z3, is that it?
@pedruzzi Since this rule was created by SIB, could we revise Case 1 where GO:0016192 was propagated?

**Comment by ValWood** [comment]
Yes, it seems I the rule was based on a phenotype annotation which although still present at SGD, is out of date. @edwong57  can confirm before proceeding to fix the mapping in case I made a mistake.

**Comment by edwong57** [comment]
@ValWood, I took a look and removed the BP annotation. It is gone from Protein2GO, but may take a little while to get into the latest SGD file.
